vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+// vitest.config.ts
+import dotenv from 'dotenv';
+dotenv.config();
```

Comment:
Let's remove this (not installed anyway) - I'll refactor the tests later to not need env variables.

Reply:
The `dotenv` is used in the `runners.ts` file. But if we can remove the requirement for tests then that would be great and I can remove this.